modules/darwin/common/flake-inputs.nix

```diff
@@ -3,7 +3,7 @@
 {
   services.telegraf.extraConfig.inputs.file =
     let
-      inputsWithDate = lib.filterAttrs (_: input: input ? lastModified) inputs;
+      inputsWithDate = lib.filterAttrs (_: input: input ? lastModified) inputs.self.inputs;
```

Comment:
Does this include basically the flake itself?

Reply:
Yes, it was including the flake.